### PR TITLE
Completed ticket 5

### PR DESCRIPTION
**Pull request by Storme419**

### CODE REVIEW

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Ideally, you should be using a separate API call to render the single article pag e- getARticleById is generally what it is called on your backend. The reason for this is that getArticles does not include the _body_ which is the text of the article, whereas getArticleById does.

You should also consider including a loading state for the single article page :)

File: src/Pages/ArticlePage.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    useEffect(() => {
+        getArticleById(id)
+        .then((res) => {
+            setCurrentArticle(res)
+            setLoading(false)
+        }).catch(console.log)
+    }, [])
+console.log(currentArticle)
+    function findTimeSince(originalDate){
```
Comment (Liam310) [inline src/Pages/ArticlePage.jsx]: Get rid of this `console.log` 🙅‍♀️ (line 18)

File: src/Pages/ArticlePage.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    return loading 
+    ? <h2 id='loading-msg'>Loading...</h2> : 
+    
+    <section className="article">
+        <h2>Article Page</h2>
+        <h3>{currentArticle.title}</h3>
+        <h4>By {currentArticle.author}</h4>
+        <img className="articleImg" src={currentArticle.article_img_url} alt={currentArticle.title} />
+        <p>Created {findTimeSince(currentArticle.created_at)}</p>
+        <p>Votes: {currentArticle.votes} </p>
```
Comment (Liam310) [inline src/Pages/ArticlePage.jsx:33]: I love what you've done with this date formatting! Very nice 👌

**Review by Liam310 — APPROVED** [review]
Looks great! Have pointed out a single rogue leftover `console.log` but you're ready to go ahead and merge away 💪

File: src/Pages/ArticlePage.jsx (hunk 1 above)
Comment (Storme419) [reply src/Pages/ArticlePage.jsx]: I knew there would be one hidden somewhere lol

**Pull request merged**